spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -205,6 +205,18 @@
         ).to be >= 0.0
       end
 
+      context 'when an enrollment does not have a unique ID' do
+        it 'generates a backwards-compatible unique ID' do
+          pending_enrollment.update(unique_id: nil)
+          stub_request_passed_proofing_results
+          expect(pending_enrollment).to receive(:usps_unique_id).and_call_original
```

Comment:
The `usps_unique_id` is documented as being deprecated. Should we want to update the job to be calling `set_unique_id` instead?

Reply:
I'd argue no. The point of this test is to make sure that the polling job continues to use backwards-compatible values for `unique_id` if the enrollment's `unique_id` value is blank. It's specifically testing [this line](https://github.com/18F/identity-idp/blob/31108ac1d5f760c0a3470116143238ca8f7e1521/app/jobs/get_usps_proofing_results_job.rb#L75) of the polling job

Reply:
So just to clarify, once we let the job run once to assign a unique ID for all records missing one, we can return to remove any reference to `#usps_unique_id` ?

Reply:
Exactly